### DISCUSSION TITLE
Building an NFT collection to fund protestors

### DISCUSSION THREAD

**Discussion by Im-Masoud** [WeAreMahsaAmini/Core#82] [Ideas]
Every revolution needs financial aid to be successful and complete. We can finance it by creating a NFT collection of photos and videos of protests in Iran and provide it to the protesters. Make an NFT of the scenes you have encountered and sell it.

**Comment by fatihany98** [comment]
I like it and it would be my pleasure if can help you with this project.

**Comment by parisamoshfegh** [comment]
This is a good idea! People will need money specially if they are going on strikes

**Comment by AnonymousMaximus2** [comment]
First of all you need to register a company and become recognised to community and they can trust and pay the money.



Please check below discussions first:


https://github.com/WeAreMahsaAmini/Core/discussions/115

**Reply by mrnobud** [reply]
That is not correct, you can establish a DAO with no real organization and company and manage the entire thing very transparently on blockchain

**Comment by nazunderground** [comment]
I stumbled upon this graphic series of different ethnic Iranians. Could be cool to launch possibly a series of these as an NFT. 
https://www.instagram.com/reel/CjBvWAbNWnX/?utm_source=ig_web_copy_link

just an idea.

**Comment by mrnobud** [comment]
Really like this idea as well, would love to help

**Comment by yznima** [comment]


**Reply by ghost** [reply]
You can use [stable diffusion's image-to-image feature](https://github.com/invoke-ai/InvokeAI/blob/main/docs/features/IMG2IMG.md) to generate art from a base image. Here are a few example results (not great but I didn't spend much time on it):
![1](https://user-images.githubusercontent.com/114612139/192875022-0b17934d-7fb3-4599-8a75-cc4b8bfaebfc.jpg)
![2](https://user-images.githubusercontent.com/114612139/192875044-81d3441f-a771-4efa-924d-1ee555b6d415.jpg)
![3](https://user-images.githubusercontent.com/114612139/192875076-eb87ccdd-a11e-487c-ae0d-6db9e0207a49.jpg)

**Comment by Ferzox** [comment]
The peaceful idea, I like it

**Reply by Ferzox** [reply]
We can help companies/stores that have closed their stores during the strike

**Comment by ghost** [comment]
@Im-Masoud @fatihany98 @mrnobud Should we start working on this? If we raise funds, we can use it to support other ideas, e.g. buying ads in NY time square to raise awareness, or pay for servers, or other stuff. I don't know much about NFTs, so I can't do this alone. I can generate art or modify art using AI, and I can code, but we need other experts such as graphic designers, marketers, ...
Should we make a discord channel?

**Comment by ghost** [comment]
Please check the following link:
https://github.com/WeAreMahsaAmini/Core/discussions/114#discussioncomment-3764485

**Comment by andishehk** [comment]
I think there are big challenges with the part that happens after the fundraising:

- Who should manage the funds and decide what that fund should be spent on?
- There are going to be consequences, like people who didn’t benefit from the funds, accusing you of taking the money for yourself.
- Any decision you make after receiving the funds on how to spend it, is going to raise disagreements.
- If we don’t know the exact goal of the fundraising, and don’t share the details, it’s improbable that people participate.

How are you planning to overcome these challenges?
I think fundraising is one of the hardest ideas to implement due to such challenges.

**Reply by Im-Masoud** [reply]
To solve this problem, we can create a DAO and people share their opinions and vote to reach a common point of view.
